src/Facade/Services/PurchaseOrderFacadeService.cs

```diff
@@ -169,14 +175,27 @@ public string GetOrderAsHtml(int orderNumber)
             return this.domainService.GetPurchaseOrderAsHtml(orderNumber);
         }
 
+        public new IResult<PurchaseOrderResource> Add(PurchaseOrderResource resource, IEnumerable<string> privileges = null, int? userNumber = null)
+        {
+            var order = this.BuildEntityFromResourceHelper(resource);
+
+            this.domainService.CreateOrder(order, privileges);
```

Comment:
This CreateOrder( ) isn't really creating the order anymore, so maybe needs a different name

Reply:
yeah I was actually kinda thinking that while I was looking at the facade earlier